src/debugpy/_vendored/pydevd/pydevd_attach_to_process/windows/compile_windows.bat

```diff
@@ -16,7 +20,7 @@ cl -DUNICODE -D_UNICODE /EHsc /Zi /O1 /W3 /LD /MD /D BITS_32 /Qspectre run_code_
 copy run_code_on_dllmain_x86.dll ..\run_code_on_dllmain_x86.dll /Y
 copy run_code_on_dllmain_x86.pdb ..\run_code_on_dllmain_x86.pdb /Y
 
-cl /EHsc /Zi /O1 /W3 /Qspectre inject_dll.cpp /link /DEBUG /OPT:REF /OPT:ICF /GUARD:CF /out:inject_dll_x86.exe
+cl /EHsc /Zi /O1 /W3 /Qspectre inject_dll.cpp /link /PROFILE /GUARD:CF /out:inject_dll_x86.exe
```

Comment:
isn't /PROFILE for profiling? I mean when you want to inject profiling stub on each symbols to gather profiling info. I thought those are usually not used unless dll is specificallly built for profiling.

Reply:
According to https://eng.ms/docs/products/apiscan/howto/preparinginput/binaries/creating_vulcan_ready_files, we have two options for APIScan to work:

```
Linking/link.exe
/profile (https://learn.microsoft.com/en-us/cpp/build/reference/profile-performance-tools-profiler)
This informs the linker to emit full fixup information so that Vulcancompletely identifies code and data cross-references. Be aware thatthis implies the following linker switches that affect codegeneration: /incremental:no /opt:ref /opt:noicf. The /opt switchescan be overridden on the command-line after /profile is used.

Optionally for non public source bases the following switches can be used instead of /profile:

/debug:full (https://learn.microsoft.com/en-us/cpp/build/reference/debug-generate-debug-info)

/debugtype:cv,fixup (https://learn.microsoft.com/en-us/cpp/build/reference/debugtype-debug-info-options)

/incremental:no (https://learn.microsoft.com/en-us/cpp/build/reference/incremental-link-incrementally)
```

So I can use the three debug switches instead of profile, It doesn't matter to me. 😄

Reply:
No read this:
https://learn.microsoft.com/en-us/cpp/build/reference/profile-performance-tools-profiler?view=msvc-170#remarks

/PROFILE implies /DEBUG:full

Reply:
surprised that /PROFILE flag is recommended to be enabled by default. I guess now it generates fast enough code to okay to be enabled by default.